lib/nylas/api.rb

```diff
@@ -150,6 +150,15 @@ def webhooks
       @webhooks ||= Collection.new(model: Webhook, api: as(client.app_secret))
     end
 
+    def free_busy(emails:, start_time:, end_time:)
+      FreeBusyCollection.new(
+        api: self,
+        emails: emails,
+        start_time: start_time.to_i,
```

Comment:
You probably already know but there aren't any timezone issues here, are there? What format does `start_time` come in (e.g. `DateTime`)?

Reply:
No issues as it's unix timestamp